python/ql/lib/semmle/python/dataflow/new/internal/LocalSources.qll

```diff
@@ -74,6 +74,8 @@ class LocalSourceNode extends Node {
     this instanceof ScopeEntryDefinitionNode
     or
     this instanceof ParameterNode
+    or
+    this instanceof IterableSequenceNode
```

Comment:
I wonder what the consequences of this is, did you test it in isolation? A new `LocalSourceNode` feels like potentially a big thing, since many predicates rely on this definition. Also, it should fit with the semantics, that a value is introduced in this place, but I guess that is arguably the case as it pops out of some container...

Reply:
Fixed :+1: